apps/mudita-center-e2e/src/page-objects/help.page.ts

```diff
@@ -7,17 +7,41 @@ import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 class HelpPage extends Page {
-  public get helpListElement(): ChainablePromiseElement<
-    Promise<WebdriverIO.Element>
-  > {
+  public get listElement(): ChainablePromiseElement<WebdriverIO.Element> {
```

Comment:
Types are now correct which is great, but to avoid boilerplate code you can delete return type from getters. If a function has one return statement and it is fairly simple it's better to skip adding a return type because under the hood  TypeScript itself will do this for you.

Reply:
`public get listElement() {
      return $('[data-testid="help-component-question"]')
  }
`